_data/glossary/cloud-provider.yaml

```diff
@@ -0,0 +1,10 @@
+id: cloud-provider
+name: Cloud Provider
+full-link: /docs/reference/kubectl/docker-cli-to-kubectl/
+tags:
+- community
+short-description: >
+  Cloud provider is a company that offers cloud computing platform that can run Kubernetes clusters.
+long-description: >
+  Cloud providers or sometime called Cloud Service Provider (CSPs) provides cloud computing platforms.  They may offer services such as Infrastructure as a Service (IaaS) or Platform as a Service (PaaS).  For Kubernetes we need Cloud Provider to host the Kubernetes cluster.  Cluster can be a sigle node or multi-node depending on the service level. 
```

Comment:
typo: "sigle" should probably be "single"

Reply:
Cloud Providers host the Kubernetes cluster and also provide services that interact with the cluster, such as LoadBalancers, StorageClasses, etc. The summary would benefit from rephrasing so that it's clear that cloud providers don't just host clusters.